sim-lib/src/lib.rs

```diff
@@ -501,18 +500,19 @@ async fn write_payment_results(
     mut receiver: Receiver<(DispatchedPayment, PaymentResult)>,
     listener: Listener,
 ) -> Result<(), SimulationError> {
-    let mut writer = WriterBuilder::new()
-        .buffer_capacity(size_of::<(DispatchedPayment, PaymentResult)>() * 5)
-        .from_path(format!(
-            "simulation_{:?}.csv",
-            SystemTime::now()
-                .duration_since(SystemTime::UNIX_EPOCH)
-                .unwrap()
-                .as_secs()
-        ))?;
+    let mut writer = WriterBuilder::new().from_path(format!(
+        "simulation_{:?}.csv",
+        SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .unwrap()
+            .as_secs()
+    ))?;
 
     let mut result_logger = PaymentResultLogger::new();
 
+    let batch_print = 5;
```

Comment:
nit: `batch_size`?

Reply:
yess

Reply:
`print_batch_size` just so we remember what we are batching?..